server/utils/errors.js

```diff
@@ -4,12 +4,14 @@ module.exports = {
 }
 
 function unauthorisedError() {
+  /** @type {any} */
```

Comment:
Can't set arbitrary properties on known types - real way to fix would be to define a type which extends `Error` and adds specific fields but leaving for now